node/src/node/Main.hs

```diff
@@ -43,30 +45,33 @@ actionWithoutWallet sscParams nodeParams =
     plugins :: ([WorkerSpec (RealMode ssc)], OutSpecs)
     plugins = updateTriggerWorker
 
-action :: SimpleNodeArgs -> Production ()
+action
+    :: ( HasConfigurations )
+    => SimpleNodeArgs
+    -> Production ()
 action (SimpleNodeArgs (cArgs@CommonNodeArgs {..}) (nArgs@NodeArgs {..})) = do
-    liftIO $ applyConfigInfo configInfo
-    giveStaticConsts $ do
-        systemStart <- CLI.getNodeSystemStart $ CLI.sysStart commonArgs
-        logInfo $ sformat ("System start time is " % shown) systemStart
-        t <- currentTime
-        logInfo $ sformat ("Current time is " % shown) (Timestamp t)
-        currentParams <- CLI.getNodeParams cArgs nArgs systemStart
-        putText $ "Running using " <> show sscAlgo
-        putText "Wallet is disabled, because software is built w/o it"
-        logInfo $ sformat ("Using configs and genesis:\n"%build) configInfo
+    whenJust cnaDumpGenesisDataPath $ CLI.dumpGenesisData
+    putStrLn $ getSharedSeed $ gdFtsSeed genesisData
```

Comment:
Probably should be deleted, because it's printed a little later anyway.

Reply:
Agree